src/backend/distributed/citus--6.0-17--6.0-18.sql

```diff
@@ -0,0 +1,22 @@
+/* citus--6.0-17--6.0-18.sql */
+
+SET search_path = 'pg_catalog';
+
+DROP FUNCTION master_add_node(nodename text, nodeport integer);
```

Comment:
At a few places in the code, we use DROP FUNCTION IF EXISTS for similar purposes. Could we use that?